requirements-dev.txt

```diff
@@ -21,3 +21,4 @@ isort
 pipdeptree
 eth-typing<2.0.0
 eth-abi<=1.2.2
+six>=1.12
```

Comment:
This looks unrelated to the rationale of the PR. Is 2/3 compatibility really a concern?

Reply:
This caused CI failures unrelated to this PR. But, I've added this line on `master` already. This diff will disappear when I rebase.

Reply:
This diff has disappeared.